src/components/_helpers/transferProps.js

```diff
@@ -1,7 +1,10 @@
 export const transferProps = ({
   children,
   className,
+  dangerouslySetInnerHTML,
   ref,
+  suppressContentEditableWarning,
```

Comment:
Why this one? The others seem reasonable to me (i.e. we use them ourselves so passing them would break things).

Maybe the items here deserve a comment (if not a MD documentation) so it doesn't fell like a random list.

Reply:
> Why this one? 

Because I think we do not want to allow users to make a HTML element within a component to have `contenteditable` attribute set if it has child nodes . There would be to many edge cases. Honestly I do not know what is there to document. By listing it is obvious that we do not allow this.

I added some docs, but I'm not sure that is there to write about the individual blacklisted props.

I think we do not want to blacklist `contenteditable` completely, because I can imagine allowing it for some components (e.g. `Badge`). Not that it is good UX, but technically there is no problem with it and UX decisions should take place on the concrete UI library based on RUI.

Reply:
I also blacklisted the `contentEditable` attribute